packages/smooth_app/lib/pages/product/nutrition_page/widgets/nutrition_facts_editor.dart

```diff
@@ -493,10 +498,10 @@ class _NutrientUnitVisibility extends StatelessWidget {
   }
 }
 
-extension NutritionTextEditionController on TextEditingController {
-  bool get isSet => text.trim() != '-';
+const String _hiddenNutrientValue = '-';
 
-  bool get isNotSet => text.trim() == '-';
```

Comment:
(good cleanup here; to maybe save someone else wondering about this line of the diff: this function was previously used by `nutrition_page_loaded.dart`, but has been removed since then, by 83244568189e579797e1dc73d45258047f4fbf0f)